languages/es-ES.js

```diff
@@ -1,169 +1,169 @@
+			RESOLVER_INVALID_ROLE: (name) => `${name} debe ser una mención o una ID de rol válida.`,
+			RESOLVER_INVALID_LITERAL: (name) => `Su opción no coincide con la siguiente posibilidad: ${name}`,
+			RESOLVER_INVALID_BOOL: (name) => `${name} debe ser 'true' o 'false'.`,
+			RESOLVER_INVALID_INT: (name) => `${name} debe ser un número entero.`,
+			RESOLVER_INVALID_FLOAT: (name) => `${name} debe ser un número.`,
+			RESOLVER_INVALID_REGEX_MATCH: (name, pattern) => `${name} debe seguir el patrón de la expresión regular \`${pattern}\``,
+			RESOLVER_INVALID_URL: (name) => `${name} debe ser un enlace URL válido.`,
+			RESOLVER_INVALID_DATE: (name) => `${name} debe ser una fecha válida.`,
+			RESOLVER_INVALID_DURATION: (name) => `${name} debe ser una duración válida.`,
+			RESOLVER_INVALID_TIME: (name) => `${name} debe ser una fecha o duración válida.`,
+			RESOLVER_STRING_SUFFIX: ' caracteres',
+			RESOLVER_MINMAX_EXACTLY: (name, min, suffix) => `${name} debe ser exactamente ${min}${suffix}.`,
+			RESOLVER_MINMAX_BOTH: (name, min, max, suffix) => `${name} debe estar entre ${min} y ${max}${suffix}.`,
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} debe ser mayor que ${min}${suffix}.`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} debe ser menor que ${max}${suffix}.`,
+			REACTIONHANDLER_PROMPT: '¿A qué página te gustaría saltar?',
+			COMMANDMESSAGE_MISSING: 'Faltan uno o más argumentos al final de la entrada.',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} es un argumento requerido.`,
+			COMMANDMESSAGE_MISSING_OPTIONALS: (possibles) => `Falta una opción requerida: (${possibles})`,
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Su opción no coincide con ninguna de las posibilidades: (${possibles})`,
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error, time) => `${tag} | **${error}** | Tienes **${time}** segundos para responder a este mensage emergente con un argumento válido. Escribe **"ABORT"** para abortar el proceso.`, // eslint-disable-line max-len
+			MONITOR_COMMAND_HANDLER_REPEATING_REPROMPT: (tag, name, time) => `${tag} | **${name}** es un argumento repetible | Tienes **${time}** segundos para responder este mensage emergente con un argumento válido. Escribe **"CANCEL"** para abortar el proceso.`, // eslint-disable-line max-len
+			MONITOR_COMMAND_HANDLER_ABORTED: 'Abortado.',
+			INHIBITOR_COOLDOWN: (remaining) => `El comando se está enfriando. Quedan ${remaining} segundos.`,
+			INHIBITOR_DISABLED: 'Este comando está desactivado.',
+			INHIBITOR_MISSING_BOT_PERMS: (missing) => `Permisos insuficientes, necesitas: **${missing}**`,
+			INHIBITOR_NSFW: 'No puedes usar comandos NSFW en este canal.',
+			INHIBITOR_PERMISSIONS: 'No tienes permiso para usar este comando.',
+			INHIBITOR_REQUIRED_CONFIGS: (settings) => `El servidor no tiene las siguientes claves ${settings.length > 1 ? 's' : ''}: **${settings.join(', ')}** y no puedo realizar esa acción.`,

```

Comment:
The following ternary gives it the `s` when it's in plural, but I just noticed there was a little bug here, 0 is also plural, so it should do "clave" when 1, and "claves" otherwise.

```suggestion
			INHIBITOR_REQUIRED_CONFIGS: (settings) => `El servidor no tiene las siguientes clave${settings.length === 1 ? '' : 's'}: **${settings.join(', ')}** y no puedo realizar esa acción.`,
```